test/tests/time_integrators/actually_explicit_euler_verification/ee-1d-linear.i

```diff
@@ -56,8 +56,6 @@
 []
 
 [BCs]
-  active = 'all'
-
```

Comment:
There were multiple instances of this in this directory. I did not mark all of them. You may want to double check the other files as well.

Reply:
👍 Got 'em